data/test_typeshed_stubs/.gitignore

```diff
@@ -0,0 +1 @@
+.*.swp
```

Comment:
We could add this to the main `.gitignore` file.

Reply:
lo and behold - it's already there. Just removing this .gitignore file.